Game.Checkers/CheckersAction.cs

```diff
@@ -1,10 +1,53 @@
-﻿namespace Game.Checkers
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Game.Checkers
 {
-    public class CheckersAction : IEquatable<CheckersAction>
+    public readonly struct Field : IEquatable<Field>
     {
-        public bool Equals(CheckersAction? other)
+        public int Col { get; }
+        public int Row { get; }
+        public Field(int col, int row)
         {
-            throw new NotImplementedException();
+            Col = col;
+            Row = row;
+        }
+        public Field(string field)
```

Comment:
dodać sprawdzenie czy parametr nie jest pusty i zawiera właściwe wartości - uniemożliwić powstanie nieprawidłowego pola

Reply:
To funkcja do łatwego pisania testów, nie do używania w kodzie. Więc nie wiem, czy jest sens tu się pilnować?

Reply:
w takim razie szkoda czasu:)